kalite/distributed/static/js/distributed/exercises/views.js

```diff
@@ -138,6 +138,14 @@ window.ExerciseView = Backbone.View.extend({
 
         // Catch the "next question" button click event -- needs to be explicit (not in "events")
         this.$("#next-question-button").click(this.next_question_clicked);
+        
+        // Workaround so that pressing enter on perseus exercises submits the answer 
```

Comment:
That's on the Backbone Events object - on a Backbone View you can define an events object that binds certain DOM events to callbacks:
```
  events: {
    "click .icon":          "open",
    "click .button.edit":   "openEditDialog",
    "click .button.delete": "destroy"
  },
```
So you could just add:
```
events: {
     "keydown .perseus-input": "methodnameforprocessingkeypress",
     "keydown #solution-area": "methodnameforprocessingkeypress"
},
```
(Hopefully this second line would then also make it work for non-perseus exercises.)